<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RAScorer</class>
 <widget class="QMainWindow" name="RAScorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>730</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>RAScorer</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,1,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="LW_Files"/>
      </item>
      <item>
       <widget class="QTextEdit" name="TE_HeadInfo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,14">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
        <item>
         <widget class="QLabel" name="LB_Reviewed">
          <property name="text">
           <string>Reviewed</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_Reviewed">
          <property name="text">
           <string>Reviewed</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QOpenGLWidget" name="GL_Xray"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4" stretch="4,0,0,2,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="LW_Score_Order"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="PB_All_Pos">
          <property name="text">
           <string>All Neg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_All_Neg">
          <property name="text">
           <string>All Pos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="PTE_Load"/>
      </item>
      <item>
       <widget class="QPushButton" name="PB_Set">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,14">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QRadioButton" name="RB_L">
            <property name="text">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RB_R">
            <property name="text">
             <string>Right</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <item>
           <widget class="QRadioButton" name="RB_JSN">
            <property name="text">
             <string>JSN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="RB_BE">
            <property name="text">
             <string>BE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="Score_Model" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Output"/>
   <addaction name="action_Input"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_Output">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="action_Input">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="action_Next">
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="action_Before">
   <property name="text">
    <string>Before</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
